bin/run-librivox.sh

```diff
@@ -1,20 +1,13 @@
 #!/bin/sh
-
 set -xe
-
-export ds_importer="librivox"
-
-export ds_train_batch_size=32
-export ds_dev_batch_size=32
-export ds_test_batch_size=32
-
-export ds_epochs=50
-export ds_display_step=5
-export ds_checkpoint_step=1
-
 if [ ! -f DeepSpeech.py ]; then
     echo "Please make sure you run this from DeepSpeech's top level directory."
     exit 1
 fi;
 
-python -u DeepSpeech.py
+python -u DeepSpeech.py \
+  --importer librivox \
+  --train_batch_size 32 \
+  --dev_batch_size 32 \
+  --test_batch_size 32 \
```

Comment:
This doesn't come from your PR, but we should probably pass `--learning_rate 0.0001` here.